pom.xml

```diff
@@ -41,13 +41,13 @@
         <starter.min.bundles.count>126</starter.min.bundles.count>
 
         <!-- versions to be replaced in the feature files -->
-        <asm.version>9.1</asm.version>
-        <jackrabbit.version>2.20.2</jackrabbit.version>
+        <asm.version>9.2</asm.version>
+        <jackrabbit.version>2.21.8</jackrabbit.version>
```

Comment:
I think we should keep using the 2.20.x stream for Jackrabbit. 2.21.x is unstable and will be released as 2.22.x

Reply:
Thanks for the clarification. Upgraded to 2.20.4 instead.